Symfony/CS/Fixer/Contrib/ShortEchoTagFixer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_OPEN_TAG_WITH_ECHO) as $index => $token) {
+            $token->setContent('<?php echo');
```

Comment:
Same comment that before. This is corrupting the token collection. you should replace the ``T_OPEN_TAG_WITH_ECHO`` token with 3 tokens: a ``T_OPEN_TAG``, a ``T_WHITESPACE`` and a ``T_ECHO`` (and ensuring there is a ``T_WHITESPACE`` after that, or inserting it otherwise).

And given it will insert new tokens, you will need to iterate backward to avoid missing elements in the iteration